recipe/meta.yaml

```diff
@@ -13,10 +13,12 @@ source:
   {{ hash_type }}: {{ hash }}
 
 build:
-  number: 0
+  number: 1
   script: 
     - python setup.py install --record=record.txt  # [not py3k]
     - python setup3.py install --record=record.txt  # [py3k]
+    - copy %PREFIX%\Lib\site-packages\pywin32_system32\*.dll %PREFIX%
```

Comment:
Thanks @msarahan, I will update to the new location then.

`pywintypes.py` has some custom code to load `pywintypesXX.dll` and one of the places it looked at was next to the Python executable, but I'm pretty sure it also looks at other places on the `%PATH%`.